### PR TITLE
Liveview screenshot tests

### LINKED ISSUE #2039: Live view tests
Add some screenshot tests for the live viewer.

### PR DESCRIPTION
**Pull request by samtygier-stfc**
### Issue

Closes #2039 

### Description

Implemented during a ensemble session.

Adds 3 screenshot tests for the live viewer, plus a small refactor.

### Testing 

This adds screenshot tests to the live viewer

### Acceptance Criteria 

Check new screenshots show on applitools, and look as expected.

### Documentation

Not needed

### CODE REVIEW

File: mantidimaging/eyes_tests/live_viewer_window_test.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        self.check_target(widget=self.imaging.live_viewer)
+
+    @mock.patch('mantidimaging.gui.windows.live_viewer.presenter.LiveViewerWindowPresenter.load_image')
+    @mock.patch('mantidimaging.gui.windows.live_viewer.model.ImageWatcher')
+    def test_rotate_operation_rotates_image(self, _mock_image_watcher, mock_load_image):
+        file_list = self._make_simple_dir(self.live_directory)
+        image_list = [Image_Data(path) for path in file_list]
+        mock_load_image.return_value = self._generate_image()
+        self.imaging.show_live_viewer(self.live_directory)
+        self.imaging.live_viewer.filter_params = {"Rotate Stack": {"params": {"angle": 90}}}
```
Comment (samtygier-stfc) [inline mantidimaging/eyes_tests/live_viewer_window_test.py:77]: If possible could we set the rotation by triggering the actual menu action. I think something like
`self.imaging.live_viewer.rotate_angles_group.actions()[1].trigger()`

**Comment by MikeSullivan7** [comment]
Could a Linux user please have a look at the `test_rotate_operation_rotates_image` screenshot on applitools? It looks and passes as expected on Windows but with the Linux screenshot it gives the error: 

> ERROR    mantidimaging.gui.windows.live_viewer.presenter:presenter.py:94 FileNotFoundError reading image: /live_dir/abc_000004.tif: [Errno 2] No such file or directory in the fake filesystem: '/proc/meminfo'

and does not display correctly: 
image

compare with Windows: 
image

However I fail to see what the difference between linux and windows is here as the test is basically the same as `test_live_view_opens_with_data`

**Comment by samtygier-stfc** [comment]
I've add a fix for `/dev/meminfo` and workaround for https://github.com/pytest-dev/pyfakefs/issues/949

File: mantidimaging/eyes_tests/live_viewer_window_test.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Copyright (C) 2023 ISIS Rutherford Appleton Laboratory UKRI
```
Comment (JackEAllen) [inline mantidimaging/eyes_tests/live_viewer_window_test.py:1]: Copyright needs to be updated to 2024

**Review by JackEAllen — CHANGES_REQUESTED** [review]
Looks Good! just need to update the copyright to 2024 now that #2049 has been merged in

**Review by JackEAllen — APPROVED** [review]

**Pull request merged**